cdap-ui/app/cdap/components/FieldLevelLineage/Fields/Fields.scss

```diff
@@ -16,57 +16,65 @@
 
 @import '~styles/variables.scss';
 
-$fields-count-color: $grey-04;
-$row-height: 32px;
-$bg-color: $grey-07;
-$outside-border: 2px solid $grey-03;
-$border-size: 1px;
-$border: $border-size solid $grey-03;
+$row-height: 29px;
+$border: 1px solid $grey-06;
 $grid-padding: 0 10px;
-$number-of-rows: 11;
 $disabled-color: $grey-04;
 $active-bg-color: $yellow-02-lighter;
-$hover-bg-color: white;
-$border-offset: 2px;
+$hover-bg-color: $grey-08;
 $no-lineage-indicator-color: $grey-01;
+$operations-color: $blue-03;
+$header-bg-color: $grey-08;
+$header-color: $grey-02;
+$operations-width: 95px;
 
 .field-level-lineage .fields-box {
-  .header {
-    display: flex;
-
-    > div { width: 50%; }
-    .fields-count { color: $fields-count-color; }
-  }
-
   .empty {
-    background-color: $bg-color;
-    height: calc((#{$row-height} + #{$border-size}) * (#{$number-of-rows + 1}) + #{$border-offset});
-    border: $outside-border;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+    text-align: center;
     font-size: 16px;
-    padding: 25px;
+    margin-top: 100px;
+  }
+
+  .search-container > .form-control {
```

Comment:
We seem to have removed `.form-control` in the markup right? Do we need this?

Reply:
removed it, because it was double applied, Reactstrap already apply `form-control` class